CRM/Core/BAO/CustomField.php

```diff
@@ -419,9 +419,6 @@ public function getOptions($context = NULL) {
     elseif ($this->data_type === 'Boolean') {
       $options = $context == 'validate' ? array(0, 1) : CRM_Core_SelectValues::boolean();
     }
-    else {
-      return FALSE;
-    }
     CRM_Utils_Hook::customFieldOptions($this->id, $options, FALSE);
     CRM_Utils_Hook::fieldOptions($this->getEntity(), "custom_{$this->id}", $options, array('context' => $context));
     return $options;
```

Comment:
don't you need to define $options = FALSE at the start to avoid e-notices?

Reply:
@eileenmcnaughton Yes I do, and now I have :-)